.firebaserc

```diff
@@ -1,6 +1,6 @@
 {
   "projects": {
-    "default": "ampstart-8dc53",
+    "default": "ampstart-staging-metalab",
```

Comment:
This probably shouldn't be changed, we don't wanna commit our Firebase stuff into Google's repo